main.py

```diff
@@ -0,0 +1,3 @@
+if __name__ == '__main__':
+    our_list = [1, 2, 3, 4, 5]
```

Comment:
cambiar el nombre a `list_1_to_5`

```suggestion
    list_1_to_5 = [1, 2, 3, 4, 5]
```